release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -293,6 +293,68 @@ This dashboard provides API server metrics, such as:
+* The `ThanosSidecarUnhealthy` alert is added. The critical alert provides notification that the Thanos sidecar is unhealthy for a specified amount of time.
+* The `NodeClockNotSynchronising` alert is updated to prevent false positives in environments that use the chrony time service (chronyd).
+* The `NodeNetworkReceiveErrs` alert is updated to ensure that the alert does not fire when only a small number of errors are reported. The rule now uses the ratio of errors to total packets instead of the absolute number of errors.
+* The `NodeNetworkTransmitErrs` alert is updated to ensure that the alert does not fire when only a small number of errors are reported. The rule now uses the ratio of errors to total packets instead of the absolute number of errors.
+
+[NOTE]
+====
+Red Hat does not guarantee backward compatibility for metrics, recording rules, or alerting rules.
+====
+
+[id="ocp-4-7-monitoring-stack-dependency-version-updates"]
+==== Version updates to monitoring stack components and dependencies
+
+{product-title} 4.7 includes version updates to the following monitoring stack components and dependencies:
+
+* The Prometheus Operator is now on version 0.44.1.
+* Thanos is now on version 0.17.2.
+* `kube-prometheus` is now on release 0.7.
+
+[id="ocp-4-7-monitoring-prometheus-operator-alertmanagerconfig-crd-"]
+==== `AlertmanagerConfig` custom resource definition in Prometheus Operator not supported
+
+Modifying Alertmanager configurations by using the `AlertmanagerConfig` custom resource definition (CRD) in the Prometheus Operator is not supported.
+
+For more information, see xref:../monitoring/configuring-the-monitoring-stack.html#support-considerations_configuring-the-monitoring-stack[Support considerations for monitoring].
+
+[id="ocp-4-7-monitoring-namespace-by-pod-and-pod-total-Grafana-dashboards"]
+==== Namespace By Pod and Pod Total Grafana dashboards
+
+The Namespace By Pod and Pod Total networking Grafana dashboards are now enabled.
+
+[id="ocp-4-7-monitoring-hwmon-hardware-telemetry-bare-metal"]
+==== HWMon data collection for hardware telemetry for bare metal clusters
+
+HWMon data collection is enabled for hardware health telemetry such as CPU temperature and fan speeds for bare metal clusters.
+
+[id="ocp-4-7-thanos-query-log-level-config"]
+==== Log level configuration field for Thanos Querier
+
+You can now configure the Thanos Querier `logLevel` field for purposes such as debugging.
+
+[id="ocp-4-7-monitoring-prometheus-config-reloader-memory-limits"]
+==== Removed `prometheus-config-reloader` memory limit for monitoring user defined projects
+
+The memory limit was removed on the Prometheus Operator `prometheus-config-reloader` container in the `openshift-user-workload-monitoring` namespace for monitoring user defined projects. This update prevents OOM kill of the `prometheus-config-reloader` container because it used more memory than the defined limit.
```

Comment:
no prometheus-config-reloader container on prometheus-operator pod, please make it more clear

Reply:
should be `config-reloader` container, the memory limit is removed since 4.7, see the memory limit in a 4.6 cluster
```prometheus-user-workload-0
Container Name: prometheus
resources: map[requests:map[cpu:6m memory:30Mi]]
Container Name: config-reloader
resources: map[limits:map[memory:25Mi] requests:map[cpu:1m memory:10Mi]]```

Reply:
@lihongyan1 @juzhao @simonpasquier PTAL at the updated wording for the `config-reloader` container memory limit. Thank you.